Makefile

```diff
@@ -10,10 +10,10 @@ download_deps:
 
 	go get -v -u -d github.com/AlekSi/pointer \
 				github.com/kisielk/errcheck \
-				github.com/golang/lint/golint \
 				github.com/stretchr/testify/... \
 				github.com/enodata/faker \
-				github.com/AlekSi/goveralls
+				github.com/AlekSi/goveralls \
+				github.com/alecthomas/gometalinter
```

Comment:
Does gometalinter have a flag to download but not install dependencies? If yes, let's use it here.